docs/troubleshooting-guide/bugs.md

```diff
@@ -1,47 +1,33 @@
 If you come across something that looks like a bug, we suggest collecting the following information to help us reproduce the issue.
 
-1. Server logs
+1. Metabase server logs
 2. Javascript console logs
 3. Can it be reproduced on the sample dataset?
 4. Your Metabase version
-5. Where Metabase is running (Docker image, AWS Elastic Beanstalk, etc)
-6. What browser version
+5. Where Metabase is running (Docker image, AWS Elastic Beanstalk, Heroku, Linux/Ubuntu, etc)
+6. Which database is used
+7. What browser version
 
 ## Helpful tidbits
 
 ### Accessing the Metabase server logs
-While you can always look for the logs Metabase leaves on your server file system (or however you collect logs), if you are logged into Metabase with an admin account, you can also access them from the drop down menu in the upper right hand corner.
+While you can always look for the logs Metabase leaves on your server file system (or however you collect logs), if you are logged into Metabase with an admin account, you can also access them from the gear drop down menu in the upper right hand corner -> Admin -> Troubleshooting -> Logs.
 
-![Logs Screenshot](images/ServerLogs.png)
+### Checking for Javascript console errors
+Metabase will show debugging information and errors to your browser's developer console.
```

Comment:
I think this should probably say "Metabase will send debugging information and errors to your browser's developer console."